docs/articles/expensify-classic/expenses/Edit-expenses.md

```diff
@@ -4,8 +4,6 @@ description: Learn how to edit expenses in Expensify, including restrictions and
 keywords: [Expensify Classic, edit expenses, merge expenses, split expenses, delete expenses]
 ---
 
-<div id="expensify-classic" markdown="1">
```

Comment:
unfortunately we still need these or the banner on the page will be removed. My branch makes this obsolete though